libcontainer/nsenter/nsexec.c

```diff
@@ -1231,23 +1231,17 @@ void nsexec(void)
 			/* Ask our parent to send the mount sources fds. */
 			if (config.mountsources) {
 				s = SYNC_MOUNTSOURCES_PLS;
-				if (write(syncfd, &s, sizeof(s)) != sizeof(s)) {
-					sane_kill(stage2_pid, SIGKILL);
```

Comment:
https://github.com/opencontainers/runc/blob/99968fc0874642300f688ce4d617a1e42ca7ce29/libcontainer/nsenter/nsexec.c#L1034-L1038

Do we need to remove 👆 line? It is unlikely, but that kill the wrong pid is bad.

Reply:
I have also noticed this one while looking into it. Obviously, we can't trust the `stage2_pid` being correct in this case.

Yes, we do (as a separate commit I guess).

Reply:
@fuweid wanna do that PR? :)

Reply:
Just point it out. 😂
I think you can handle it in this pr. 😄

Reply:
@fuweid sure, added. Thanks!